### ISSUE TITLE
Bluetooth issue

### ISSUE THREAD

**Issue by mottoyin** [baughmann/designaire-z390-intel-i9-9900k-opencore#16]
**Describe the bug**

**OpenCore Logs**
To figure out how to enable logging, see the relevant [section of the OpenCore docs](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/debug.html#file-swaps).

```
Is it possible to add AirportBrcmFixup.kext into the master trunk? I use almost same as your platform except the blutooth and wifi card. My module is 94360cs2 which could not be identified in Big Sur and this bothers me a lot. Appreciated if you could update the package and the plst. Thanks a lot you guys did a good job :)
```

**Expected behavior**

**Screenshots**

**Hardware Specifications (please complete the following information):**
 - CPU: [Intel Core i9 9900k]
 - GPU: [AMD Radeon VII]
 - Motherboard: [Designaire Z390]
 - Storage: [make, model, and partition scheme--i.e., "APFS"]

**Additional context**

**Comment by baughmann** [comment]
@mottoyin It looks like you did not follow the bug reporting procedures. You used the template but did not fill it out correctly.

Which OS are you on? Big Sur or Catalina? If you're still on Catalina and need the AirportBrcm kexts, then please use [this release](https://github.com/baughmann/designaire-z390-intel-i9-9900k-opencore/releases/tag/1.0-oc_0.6.0) as I [described in the readme.](https://github.com/baughmann/designaire-z390-intel-i9-9900k-opencore#note-the-latest-release-is-for-big-sur-may-not-work-with-catalina-anymore-see-the-previous-release-if-you-are-still-on-catalina)

**Comment by baughmann** [comment]
I may look into re-integrating the AirportBrcm kexts, but since they're not necessary for my build anymore, and this repo only officially supports builds like mine, I also may not.

However, if you need the kexts, just integrate them yourself. It's pretty easy. Just:
1. [Download](https://github.com/acidanthera/airportbrcmfixup/releases) the kexts that you need
2. Add them to your `config.plist` and enable them, as [described in the OpenCore docs](https://dortania.github.io/OpenCore-Install-Guide/config.plist/#adding-your-ssdts-kexts-and-firmware-drivers)

**Comment by baughmann** [comment]
I'm gonna go ahead and close this now since it's not an issue with the build, but feel free to comment here if you have questions about integrating the kexts.

**Comment by ryanpsims73** [comment]
Yeah weird, same kexts, used your plist, same hardware, no Bluetooth at all. Did a rebuild of the kext cache as well.

**Comment by baughmann** [comment]
@ryanpsims73 What happens if you include the AirportBrcm kexts? Does it fix it?

**Comment by baughmann** [comment]
According to [the docs](https://dortania.github.io/Wireless-Buyers-Guide/Airport.html#socketed-cards) our WiFi/BT card (the BCM94360CD) should be supported natively.

Previously I had to use the AirportBrcm fixes because I didn't want to remove my integrated card on the MoBo. I've got the unsubstantiated suspicion that in BIOS version F9i (what I'm on now) turning off this card in the BIOS is what is allowing me to run the Fenvi without my custom `USBPorts.kext` and without any of the BRCM fixes.
